ubuntu_image/parser.py

```diff
@@ -411,6 +412,10 @@ def parse(stream_or_string):
                     offset = MiB(1)
                 else:
                     offset = last_offset
+            if (size % sector_size) != 0 or (offset % sector_size) != 0:
+                warn('Partition size/offset need to be a multiple of sector '
+                     'size ({}).  The size/offset will be rounded up to the '
+                     'nearest sector.'.format(sector_size))
```

Comment:
Hey! I didn't write an explicit test for this since I thought it's already tested through the test_small_partition_size_and_offset() you made. But when I look at it, it actually only tests the offset being rounded! Should I add an explicit test that checks both sizes and offsets?